tests/unit/serve/executors/test_decorators.py

```diff
@@ -2,9 +2,10 @@
 
 import pytest
 
+from jina import Executor
```

Comment:
this test should not import `Executor` since this is unit test only for the decorator